modules/Bio/EnsEMBL/Analysis/Hive/Config/OtherFeatureDb.pm

```diff
@@ -0,0 +1,583 @@
+    'refseq_mrna_ftp_path'    => $self->o('refseq_base_ftp').'/#assembly_refseq_accession#_#assembly_name#_rna.fna.gz',
+    'refseq_report_ftp_path' => $self->o('refseq_base_ftp').'/#assembly_refseq_accession#_#assembly_name#_assembly_report.txt',
+##################################
+# Memory settings for the analyses
+##################################
+    'default_mem'          => '900',
+    'genblast_mem'         => '1900',
+    'genblast_retry_mem'   => '4900',
+    'genewise_mem'         => '3900',
+    'genewise_retry_mem'   => '5900',
+    'refseq_mem'           => '9900',
+    'projection_mem'       => '1900',
+    'layer_annotation_mem' => '3900',
+    'genebuilder_mem'      => '1900',
+
+
+########################
+# db info
+########################
+    'cdna_db' => {
+      -dbname => $self->o('dbowner').'_'.$self->o('production_name').'_cdna_'.$self->o('release_number'),
+      -host   => $self->o('cdna_db_server'),
+      -port   => $self->o('cdna_db_port'),
+      -user   => $self->o('user'),
+      -pass   => $self->o('password'),
+      -driver => $self->o('hive_driver'),
+    },
+
+    'projection_source_db' => {
+      -dbname => $self->o('projection_source_db_name'),
+      -host   => $self->o('projection_source_db_server'),
+      -port   => $self->o('projection_source_db_port'),
+      -user   => $self->o('user_r'),
+      -pass   => $self->o('password_r'),
+      -driver => $self->o('hive_driver'),
+    },
+
+    'refseq_db' => {
+      -dbname => $self->o('dbowner').'_'.$self->o('production_name').'_refseq_'.$self->o('release_number'),
+      -host   => $self->o('refseq_db_server'),
+      -port   => $self->o('refseq_db_port'),
+      -user   => $self->o('user'),
+      -pass   => $self->o('password'),
+      -driver => $self->o('hive_driver'),
+    },
+
+    'production_db' => {
+      -host   => $self->o('production_db_server'),
+      -port   => $self->o('production_db_port'),
+      -user   => $self->o('user_r'),
+      -pass   => $self->o('password_r'),
+      -dbname => 'ensembl_production',
+      -driver => $self->o('hive_driver'),
+    },
+
+    'otherfeatures_db' => {
+      -dbname => $self->o('dbowner').'_'.$self->o('production_name').'_otherfeatures_'.$self->o('release_number'),
+      -host   => $self->o('otherfeatures_db_server'),
+      -port   => $self->o('otherfeatures_db_port'),
+      -user   => $self->o('user'),
+      -pass   => $self->o('password'),
+      -driver => $self->o('hive_driver'),
+    },
+
+  };
+}
+
+sub pipeline_create_commands {
+    my ($self) = @_;
+
+    return [
+    # inheriting database and hive tables' creation
+      @{$self->SUPER::pipeline_create_commands},
+    ];
+}
+
+
+sub pipeline_wide_parameters {
+  my ($self) = @_;
+
+
+  return {
+    %{$self->SUPER::pipeline_wide_parameters},
+    wide_ensembl_release => $self->o('ensembl_release'),
+  }
+}
+
```

Comment:
wide_ensembl_release is not needed anywhere and can be removed.
Because both methods are only calling the parent method they can be removed. They could stay as it can make life easier if someone needs to add a pipeline wide parameters or create a directory at the beginning of the pipeline

Reply:
wide_ensembl_release removed.
For the methods I would like to delete them in order to clean as much as possible, we can always decide to add them later